source/blog/2015-02-26-digital-healthcare-roundtable-session.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: "blog"
+date: "2015-02-26 13:04:00 UTC"
+published: true
+title: "Digital Healthcare Roundtable Session"
+author: "Graeme McCubbin"
+tags:
+- Agile
+- Innovation
+---
+
+Wednesday 18th February saw the Digital Healthcare Roundtable Session take place at Unboxed Consulting.  
+
+
+ The session brought together a handpicked and diverse group of healthcare experts and leaders, with the aim to start a revolution in digital healthcare. Ideas were explored and exchanged about how we can do more to improve public health with digital technology, in this one-off discussion.  
+  
+
+ [Martyn Evans](http://www.unboxedconsulting.com/people/martyn-evans) and [Carrie Bedingfield](http://www.unboxedconsulting.com/people/carrie-bedingfield) both attended from Unboxed Consulting, with Carrie chairing the session.  
+  
+
+ ![healthcare0](http://i1291.photobucket.com/albums/b548/grammccram/DSC01779\_zpsd4fmmo16.jpg)
+
+With everyone comfortably seated, the discussion was opened, addressing the initial questions:  
+
+ • _Why is healthcare still so clunky?_  
```

Comment:
Could catch, actually a whitespace was spread all over the article, so I've removed it from all places. 
Also the example you've posted is no longer _italic_ so it's been fixed as well.